src/braket/default_simulator/openqasm/program_context.py

```diff
@@ -384,7 +396,7 @@ def __init__(self):
         self.scope_manager = ScopeManager(self)
         self.inputs = {}
         self.num_qubits = 0
-        self.circuit = Circuit()
+        self.circuit = program
```

Comment:
I did update this because there are so many test cases which access the attribute circuit. And there might be users who are accessing it as circuit.